src/client/src/models/Scene.js

```diff
@@ -16,9 +16,8 @@ const Scene = {
     }
 
     if(!localStorage.getItem('file-name')) {
-      localStorage.setItem('file-name', JSON.stringify({
-        'file-name': ''
-      }))
+      localStorage.setItem('file-name', JSON.stringify('Scenes'))
```

Comment:
Why are you stringifying a string? :)

Reply:
You're not just turning a string into a string here, you're turining a string into a json formatted string. If you remove the json stringify, it will most likely error out.